pyface/image/image.py

```diff
@@ -0,0 +1,1336 @@
+#  Copyright (c) 2005, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in enthought/LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+#  Author: David C. Morrill
+#  Date:   11/03/2007
+
+""" Defines the ImageLibrary object used to manage Pyface image libraries.
+"""
+
+import sys
+from os import (environ, listdir, remove, stat, makedirs, rename, access,
+                R_OK, W_OK, X_OK)
+from os.path import (join, isdir, isfile, splitext, abspath, dirname,
+                     basename, exists)
+from stat import ST_MTIME
+from platform import system
+from zipfile import is_zipfile, ZipFile, ZIP_DEFLATED
+import datetime
+import time
+from thread import allocate_lock
+from threading import Thread
+
+from traits.api import (HasPrivateTraits, Property, Str, Int, List, Dict,
+                        File, Instance, Bool, Undefined, TraitError, Float,
+                        Any, cached_property)
+from traits.trait_base import get_resource_path, traits_home
+
+from pyface.api import ImageResource
+from pyface.resource_manager import resource_manager
+from pyface.resource.resource_reference import (ImageReference,
+                                                ResourceReference)
+from pyface.ui_traits import HasMargin, HasBorder, Alignment
+
+#---------------------------------------------------------------------------
+#  Constants:
+#---------------------------------------------------------------------------
+
+# Standard image file extensions:
+ImageFileExts = ( '.png', '.gif', '.jpg', 'jpeg' )
```

Comment:
No, I'm planning to do an autopep8-ing of Pyface at some point, but it shouldn't be part of this PR.